injected/src/content-feature.js

```diff
@@ -247,16 +249,16 @@ export default class ContentFeature extends ConfigFeature {
             if (typeof descriptorProp === 'function') {
                 const addDebugFlag = this.addDebugFlag.bind(this);
                 const wrapper = new Proxy(descriptorProp, {
-                    apply(_, thisArg, argumentsList) {
+                    apply(target, thisArg, argumentsList) {
                         addDebugFlag();
-                        return Reflect.apply(descriptorProp, thisArg, argumentsList);
+                        return target.apply(thisArg, argumentsList);
                     },
                 });
                 descriptor[k] = wrapToString(wrapper, descriptorProp);
             }
         });
 
-        return defineProperty(object, propertyName, descriptor);
+        return defineProperty(object, String(propertyName), /** @type {any} */ (descriptor));
```

Comment:
Doesn't the above remove all checking? I think I'd rather have ignores through the codebase and/or explicitly passing the type inwards.

Reply:
Makes sense -- I basically just copied and pasted this bit from the PoC PR. See what you think of the fix.